### PR TITLE
Hello airbrake

### PR DESCRIPTION
**Pull request by mtsmfm**
$ rails g airbrake

### CODE REVIEW

File: config/newrelic.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+  app_name: daimon-news-multi-tenant
+
+  # To disable the agent regardless of other settings, uncomment the following:
+  # agent_enabled: false
+
+  # Logging level for log/newrelic_agent.log
+  log_level: info
+
+  send_data_on_exit: false
```
Comment (tricknotes) [inline config/newrelic.yml:27]: Maybe needs comment why this should be `false`?

Comment (mtsmfm) [inline config/newrelic.yml:27]: :raising_hand:

File: config/initializers/airbrake.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  # environments. By default, it's not set.
+  # https://github.com/airbrake/airbrake-ruby#environment
+  c.environment = Rails.env
+
+  # Setting this option allows Airbrake to filter exceptions occurring in
+  # unwanted environments such as :test. By default, it is equal to an empty
+  # Array, which means Airbrake Ruby sends exceptions occurring in all
+  # environments.
+  # https://github.com/airbrake/airbrake-ruby#ignore_environments
+  c.ignore_environments = %w(test)
```
Comment (tricknotes) [inline config/initializers/airbrake.rb]: Is `development` not ignored?

Comment (mtsmfm) [inline config/initializers/airbrake.rb]: airbrake gem is grouped as production, so this setting is not necessary :put_litter_in_its_place: :scissors:

Comment (tricknotes) [inline config/initializers/airbrake.rb]: Ah, right!

**Comment by tricknotes** [comment]
これはオーサムです :dango:

**Pull request merged**